docs/userGuide/syntax/boxes.mbdf

```diff
@@ -45,6 +45,47 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
 </variable>
 </include>
 
+**The built in types can be colored.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+<box>
+    default
+</box>
+<box type="info" theme="primary">
```

Comment:
Let's add the `theme` prop to the summary table down below too

Reply:
Got it!